### ISSUE TITLE
Dark theme

### ISSUE THREAD

**Issue by rrhuffy** [LibreHardwareMonitor/LibreHardwareMonitor#116]
What about adding a dark theme?
So instead of this:
![2019-08-13 17_56_53-Window](https://user-images.githubusercontent.com/14236692/62956931-185e1680-bdf4-11e9-9eb3-0253fdaa1c10.png)
It can look like this:
![2019-08-13 17_56_53-Window-negative-bright](https://user-images.githubusercontent.com/14236692/62956958-26139c00-bdf4-11e9-9bb1-ee4c97bd70ca.png)
(Just to avoid confusion - it is a manipulated photo, not a compiled PoC ;) )

**Comment by 123Inge** [comment]
This is needed. Please

**Comment by KyodaiKen** [comment]
I am working on a HTML5 based chart dashboard for the plots in #148 but it now tempts me to design a new UI using WPF... But the time ... :C

**Comment by 123Inge** [comment]
I might be the only one that does not care about the plotting in dark mode. Only want the 'main' window in dark mode.

**Comment by KyodaiKen** [comment]
As I said it is possible to do this by changing the UI to WPF and then using themes. I wonder what the main developers of LHM think about this idea.

**Comment by rvleic01** [comment]
Please add a dark mode, will make this so much better to have up on a second screen

**Comment by hl2guide** [comment]
Gotta second that! 👍

**Comment by xcasxcursex** [comment]
bump :)

**Comment by xRayUnlimited** [comment]
Does anyone know where the colors are contained? I'm looking at UI> PlotPanel.cs, which is the only place I've found Rgb color values referenced. Once I find the objects, I should be able to theme it/change the colors.

**Comment by xcasxcursex** [comment]
> Does anyone know where the colors are contained? I'm looking at UI> PlotPanel.cs, which is the only place I've found Rgb color values referenced. Once I find the objects, I should be able to theme it/change the colors.

Nowhere in particular: https://github.com/LibreHardwareMonitor/LibreHardwareMonitor/search?q=color

**Comment by MRog40** [comment]
https://github.com/LibreHardwareMonitor/LibreHardwareMonitor/pull/847

**Comment by Tamarindo94** [comment]
there is need

**Comment by 314159DD** [comment]
is there an update on this? Is it planned to implement dark mode? 
Makes it more eye friendly to be left on a second monitor at night.
Please implement a dark theme.

**Comment by KyodaiKen** [comment]
I have moved on to HWiNFO because my last pull request was modified and has been broken (It was support for high core count Threadrippers). I am also running a 13900KS now. I now use remotehwinfo to plot graphs on a laptop screen on the wall using a Raspberry Pi 4. So, unfortunately there will be no HTML5 dashboard for LHM anymore from my side. But it's very scuffed. When I find time I can put this onto Github as an addon to HWiNFO.

**Comment by oscarbg** [comment]
+1

**Comment by camoody1** [comment]
JayzTwoCents requested a Dark Mode today on his latest video where he talks about your software.
https://www.youtube.com/watch?v=sX2ThUWNXio

**Comment by progenitor-amborella** [comment]
> JayzTwoCents requested a Dark Mode today on his latest video where he talks about your software.
> 
> https://www.youtube.com/watch?v=sX2ThUWNXio

Indeed, the issues J2¢ mentioned should get posted / bumped to avoid duplication issues, as people that don't understand much about git will likely flood this repo. 

Great to have them! But certainly would like to make it easier on the maintainers.
